src/cpp/FilesInformation/File.cpp

```diff
@@ -28,3 +28,7 @@ bool File::exists() const {
     std::fstream file(_filepath.c_str());
     return file.good();
 }
+
+bool File::is_text_file() const {
+    return _extension == "txt";
```

Comment:
So we will not be able to work with. e.g. source files in different languages, or Markdown, or JSON, or any other file that is not .txt ?
Probably the implementation should be something like this:

1. Open file in binary mode
2. Read data from file (either entire file, or up to some limit)
3. If it contains unreadable characters - it is considered binary
4. Otherwise it is text

There may be complications if we are working with Unicode - ,maybe for now ignore this?
Or maybe there is some library that already implements this feature in correct and efficient way